wdl/transforms/new-base/src/main/scala/wdl/transforms/base/ast2wdlom/AstToWorkflowDefinitionElement.scala

```diff
@@ -26,26 +27,65 @@ object AstToWorkflowDefinitionElement {
                               sourceLocation: Option[SourceFileLocation],
                               bodyElements: Vector[WorkflowBodyElement]) = {
 
-    val inputsSectionValidation: ErrorOr[Option[InputsSectionElement]] = validateSize(bodyElements.filterByType[InputsSectionElement], "inputs", 1)
-    val outputsSectionValidation: ErrorOr[Option[OutputsSectionElement]] = validateSize(bodyElements.filterByType[OutputsSectionElement], "outputs", 1)
+    val inputsSectionValidation: ErrorOr[Option[InputsSectionElement]] = for {
+      inputValidateElement <- validateSize(bodyElements.filterByType[InputsSectionElement], "inputs", 1): ErrorOr[Option[InputsSectionElement]]
+      _ <- checkIfStdInputExist(inputValidateElement, StdoutElement, "stdout")
+      _ <- checkIfStdInputExist(inputValidateElement, StderrElement, "stderr")
```

Comment:
I would love to rename this function to a more generic `checkDisallowedElement` instead of including `Std...` in the name.

Looking at this call site alone (without viewing the function definition) it is not clear whether we are checking for `"stdout"` because we disallow it, or because we require it, or we're just curious ...

( applies here and elsewhere )